src/modules/menu/components/search/SearchBar.vue

```diff
@@ -130,7 +141,8 @@ const focusSearchInput = () => {
             @input="updateSearchQuery"
             @focus="onSearchInputFocus"
             @keydown.down.prevent="goToFirstResult"
-            @keydown.esc.prevent="closeSearchResults"
+            @keydown.esc.prevent="toggleResults"
```

Comment:
The escape toggle is only working when focus is on the search bar and not when it is focused on a search bar.

Reply:
Yes I know but this would make the code a bit more complicated as we would need to pass the event up a few hierarchy and we would also need to correctly manage focus.

Reply:
@LukasJoss  Actually on old viewer the ESC was clearing the input which make more sense what do you think ?

Reply:
I prefer the toggle but its up to you